discovery-infra/tests/test_e2e_install.py

```diff
@@ -15,28 +15,34 @@
 class TestInstall(BaseTest):
 
     @pytest.fixture
-    def new_cluster_configuration(self, request):
-        # Overriding the default BaseTest.new_cluster_configuration fixture to set the openshift version.
+    def new_cluster_configuration(self, request: FixtureRequest):
+        # Overriding the default BaseTest.new_cluster_configuration fixture to set custom configs.
         config = ClusterConfig()
 
-        with suppress(FixtureLookupError):
-            # Resolving the param value.
-            version = request.getfixturevalue("openshift_version")
-            config.openshift_version = version
+        for fixture_name in ["openshift_version", "network_type", "is_static_ip"]:
+            with suppress(FixtureLookupError):
+                setattr(config, fixture_name, request.getfixturevalue(fixture_name))
 
         return config
 
-    @JunitTestSuite()
-    @pytest.mark.parametrize("openshift_version", get_available_openshift_versions())
-    def test_install(self, cluster, openshift_version):
-        cluster.prepare_for_installation()
-        cluster.start_install_and_wait_for_installed()
+    # @JunitTestSuite()
+    # @pytest.mark.parametrize("openshift_version", get_available_openshift_versions())
+    # def test_install(self, cluster, openshift_version):
```

Comment:
So you wanted this as ``def test_versions`` and the other function as ``test_networking``?

Reply:
Irrelevant to this PR.
The new function is `test_networking`